src/components/login.jsx

```diff
@@ -82,27 +94,54 @@ class Login extends Component {
           <form onSubmit={this.handleSubmit}>
             <LoginInput
               name="username"
+              type="text"
               value={this.state.account.username}
               label="Username"
               onChange={this.handleInputChange}
               error={this.state.errors.username}
             />
-
             <LoginInput
               name="password"
+              type="password"
               value={this.state.account.password}
               label="Password"
               onChange={this.handleInputChange}
               error={this.state.errors.password}
             />
-
+            <Link to="/recovery">
+              Forgot your password? <span className="sr-only">(current)</span>
+            </Link>
+            <br />
+            <br />
             <button
               type="submit"
               disabled={this.validate()}
-              className="btn btn-primary"
+              class="btn btn-primary"

```

Comment:
If I remember correctly, JSX should NOT have any class tags, as it is already a keyword for components. Please replace this with className